node/src/Pos/Configuration.hs

```diff
@@ -148,8 +155,18 @@ mdNoBlocksSlotThreshold :: (HasNodeConfiguration, Integral i) => i
 mdNoBlocksSlotThreshold = fromIntegral . ccMdNoBlocksSlotThreshold $ nodeConfiguration
 
 ----------------------------------------------------------------------------
--- Transactions resubmition
+-- Wallet parameters
 ----------------------------------------------------------------------------
 
 pendingTxResubmitionPeriod :: HasNodeConfiguration => Second
 pendingTxResubmitionPeriod = fromIntegral . ccPendingTxResubmissionPeriod $ nodeConfiguration
+
+-- | If 'True', some dangerous endpoints, like one which resets wallet state,
```

Comment:
The code is [present](https://github.com/input-output-hk/cardano-sl/blob/master-release-candidate/node/src/Pos/Configuration.hs#L164-L172), no clue why GH is showing this in green.